packages/portal-proto/src/components/Table/VerticalTable.tsx

```diff
@@ -267,11 +305,15 @@ function VerticalTable<TData>({
                         }
                         colSpan={header.colSpan}
                       >
-                        <button className="flex items-center font-heading">
-                          {flexRender(
-                            header.column.columnDef.header,
-                            header.getContext(),
+                        <button
+                          className="flex items-center font-heading"
+                          onClick={handleSorting}
+                          onKeyDown={createKeyboardAccessibleFunction(
+                            handleSorting,
                           )}
+                          aria-label={ariaText}
+                        >
+                          {headerName}
 
```

Comment:
i would consider removing all of the above code changes and just add hidden text in the button directly 
```suggestion
<span className="sr-only">test</span>
```
where test is dynamic text based on the sorting ability and status of a column

Reply:
I agree with Shawn that this is not the most straightforward approach. I think his suggestion is good or I've also seen putting an aria-label on the sorting button describing the sorting state as an approach: https://inclusive-components.design/data-tables/#sortabletables